src/cmap/connection.ts

```diff
@@ -439,7 +439,7 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
         zlibCompressionLevel: this.description.zlibCompressionLevel
       });
 
-      if (options.noResponse) {
+      if (options.noResponse || ('moreToCome' in message && message.moreToCome)) {
```

Comment:
I decided to leave removing `noResponse` out of this PR since it would require editing of the public API for `CommandOperationOptions` and `RunAdminCommandOperation` which may be a breaking change. I suggest if we want this change, we should defer creating a separate ticket and working on it there.

Reply:
Can you then deprecate the `noResponse` option on the options then with a comment and link to a ticket to remove it please?

Reply:
Yep, will do!